Task Liverpool/src/main/java/pages/BasePage.java

```diff
@@ -0,0 +1,53 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+import java.util.List;
+
+public class BasePage {
+
+    private WebDriver webDriver;
+    private WebDriverWait wait;
+
+    protected BasePage(WebDriver webDriver){
+        this.webDriver = webDriver;
+        wait = new WebDriverWait(webDriver, Duration.ofSeconds(10,1));
+    }
+
+    protected WebDriver getWebDriver(){
+        return webDriver;
+    }
+
+    protected WebElement findElement(By locator){
+        wait.until(ExpectedConditions.presenceOfElementLocated(locator));
```

Comment:
the wait.until method already return the WebElement, there is no need to find it again

Reply:
fixed. This can't be done for findElements right? because there we would need a List right?

Reply:
I noticed that yes, it can also be done for findElements, I updated the method.